scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -213,6 +234,10 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
                             help='Option to fail the project export process on any failed asset during asset building (applicable if --should-build-assets is true)')
         parser.add_argument('-sl', '--seedlist', type=pathlib.Path, dest='seedlist_paths', action='append',
                                 help='Path to a seed list file for asset bundling. Specify multiple times for each seed list.')
+        parser.add_argument('-sf', '--seedfile', type=pathlib.Path, dest='seedfile_paths', action='append',
```

Comment:
Can you add a tool config argument that defaults to profile for tool binaries, the same way the --config option above is specified
```
   parser.add_argument('-toolcfg', '--tool-config', type=str, default='profile', choices=['release', 'profile', 'debug'],
                            help='The CMake build configuration to use when building tool binaries.  Defaults to profile.')
```